Library/Formula/cyassl.rb

```diff
@@ -14,39 +12,55 @@ class Cyassl < Formula
     sha1 "c5de09829f89696a73a8f3818bbf413eae99e5ac" => :mountain_lion
   end
 
+  option "without-check", "Skip compile-time tests. Saves 20 seconds."
+  option "with-aesni", "Build with support for AES-NI, which most >i5 CPUs support"
```

Comment:
That's what I wanted to do, but I wasn't entirely sure if Homebrew could detect the AES-NI status of the CPU. Looks like ` /usr/sbin/sysctl -n hw.optional.aes ` works. Do I need to call that directly in formula, or can I do something like ` if !MacOS_prefer_aes? ` I presume that's not actually working code, but in general.